composer.json

```diff
@@ -10,11 +10,12 @@
         }
     ],
     "require": {
-        "php": "^7.0"
+        "php": ">=7.1",
+        "ext-zlib": "*"
```

Comment:
The `gz*` functions require this extension and it should check the extension is loaded during `composer install` command executing.